Patch-Source: https://github.com/Kitware/VTK/commit/1233ceec.patch
Patch-Source: https://github.com/Kitware/VTK/commit/4d8283bc.patch
--
From 1233ceec268d5366c66f5e79786ec784042b591b Mon Sep 17 00:00:00 2001
From: Laurent Rineau <laurent.rineau@cgal.org>
Date: Tue, 17 Jan 2023 16:18:53 +0100
Subject: [PATCH] Add #include <cstdint> to compile with gcc13

The `vtkSEPReader` was introduced by MRs !4909 (from my former
collaborator Maxime) and !4938. Then it was highly modified by
!7516. The later MR is the one that introduced the uses of
`std::uint8_t` and `std::uint32_t`.

Those types needs the inclusion of `<cstdint>`.
---
 IO/Image/vtkSEPReader.h | 5 +++--
 1 file changed, 3 insertions(+), 2 deletions(-)

diff --git a/IO/Image/vtkSEPReader.h b/IO/Image/vtkSEPReader.h
index a7d8aad1510..37d0c44d18c 100644
--- a/IO/Image/vtkSEPReader.h
+++ b/IO/Image/vtkSEPReader.h
@@ -25,8 +25,9 @@
 #include "vtkImageAlgorithm.h"
 #include "vtkNew.h" // for ivars
 
-#include <array>  // for std::array
-#include <string> // for std::string
+#include <array>   // for std::array
+#include <cstdint> // for std::uint8_t and std::uint32_t
+#include <string>  // for std::string
 
 namespace details
 {
--
From 4d8283bcb460ff60d8df1b7b67b5db19876a5ca3 Mon Sep 17 00:00:00 2001
From: Laurent Rineau <laurent.rineau@cgal.org>
Date: Thu, 23 Feb 2023 15:34:56 +0100
Subject: [PATCH] More #include <cstdint> to compile with gcc13

---
 IO/PIO/PIOData.h                                      | 1 +
 Rendering/Matplotlib/vtkMatplotlibMathTextUtilities.h | 3 ++-
 2 files changed, 3 insertions(+), 1 deletion(-)

diff --git a/IO/PIO/PIOData.h b/IO/PIO/PIOData.h
index 8eddfb5b927..2a27c6b92e3 100644
--- a/IO/PIO/PIOData.h
+++ b/IO/PIO/PIOData.h
@@ -3,6 +3,7 @@
 
 #include "vtkABINamespace.h"
 
+#include <cstdint>
 #include <fstream>
 #include <iostream>
 #include <list>
diff --git a/Rendering/Matplotlib/vtkMatplotlibMathTextUtilities.h b/Rendering/Matplotlib/vtkMatplotlibMathTextUtilities.h
index cbccd727ddf..d3e233a9394 100644
--- a/Rendering/Matplotlib/vtkMatplotlibMathTextUtilities.h
+++ b/Rendering/Matplotlib/vtkMatplotlibMathTextUtilities.h
@@ -49,7 +49,8 @@
 #include "vtkMathTextUtilities.h"
 #include "vtkRenderingMatplotlibModule.h" // For export macro
 
-#include <vector> // for std::vector
+#include <cstdint> // for std::uint64_t
+#include <vector>  // for std::vector
 
 struct _object;
 typedef struct _object PyObject;
